Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -2166,8 +2182,7 @@ static bool isInRecoveryOS()
     clearAllEditCommands();
 
     if (!m_isPreparingToUnparentView) {
-        NSWindow *stopObservingWindow = m_targetWindowForMovePreparation.get() ?: [m_view window];
-        [m_windowVisibilityObserver stopObserving:stopObservingWindow];
+        [m_windowVisibilityObserver stopObserving];
```

Comment:
Is this necessary? It seems `- (void)startObserving:(NSWindow *)window` also invokes `stopObserving`